drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -1,7 +1,6 @@
 /*
  *
- * Copyright (c) 2019 Linaro Limited.
- * Copyright (c) 2020 Jeremy LOCHE
+ * Copyright (c) 2021 Blue Clover Devices
```

Comment:
I might be the one to blame here:
Please keep existing copyrights here (you're not adding a new file)

Reply:
That's my bad, I should have realized that from the contribution guidelines.

Fixed here and in the adc tests.